### PR TITLE
add footer; switch to key binding style, so they are displayed

### PR DESCRIPTION
**Pull request by mgeplf**

### CODE REVIEW

File: bluepyentity/app/explorer.py [hunk 1]
```diff
@@ -199,27 +200,42 @@ def leaves(adt, agg):
...
 
-    def on_key(self, event: events.Key) -> None:
-        """Manage key pressed events."""
-        if event.character == "f":
-            self._follow_cmd = not self._follow_cmd
-            if self._follow_cmd:
-                self.display_hints()
-            else:
-                self.hide_hints()
+    BINDINGS = [
```
Comment (jdcourcol) [inline bluepyentity/app/explorer.py:203]: I did not know that one !

File: bluepyentity/app/explorer.py [hunk 2]
```diff
@@ -199,27 +200,42 @@ def leaves(adt, agg):
...
-                self._refresh_all(url)
+        if len(self._previous_urls) > 1:
+            self._previous_urls.pop()
+            url = self._previous_urls[-1]
+            self._invalidate_root()
+            self._refresh_all(url)
+
+    async def action_quit(self) -> None:
+        """Quit the app"""
+        if self._follow_cmd:
```
Comment (jdcourcol) [inline bluepyentity/app/explorer.py:231]: that is a bit annoying to have that for all the other keys. but that is fine for now.

File: bluepyentity/app/explorer.py [hunk 3]
```diff
@@ -199,27 +200,42 @@ def leaves(adt, agg):
...
-        if event.character == "f":
-            self._follow_cmd = not self._follow_cmd
-            if self._follow_cmd:
-                self.display_hints()
-            else:
-                self.hide_hints()
+    BINDINGS = [
+        ("f", "follow", "Open a link"),
+        ("b", "back", "Back"),
+        ("q", "quit", "Quit"),
```
Comment (jdcourcol) [inline bluepyentity/app/explorer.py:206]: for some reason, it displays the key in uppercase. I guess this is a textual issue.

**Review by jdcourcol — APPROVED** [review]

File: bluepyentity/app/explorer.py (hunk 3 above)
Comment (mgeplf) [reply bluepyentity/app/explorer.py:206]: yeah :(

File: bluepyentity/app/explorer.py (hunk 1 above)
Comment (mgeplf) [reply bluepyentity/app/explorer.py:203]: yeah, I added the `Footer()` and discovered it that way; I'm going to move the bindings to the top of the file.

**Pull request merged**